presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleTester.java

```diff
@@ -65,9 +66,42 @@ public RuleAssert assertThat(Rule rule)
         return new RuleAssert(metadata, costCalculator, session, rule, transactionManager, accessControl);
     }
 
+    public RuleAssert assertThat(RuleSet rules)
```

Comment:
`assertThat(Rule)` is more important than `assertThat(RuleSet)` so let me leave it as is.